lib/Map/LeafletVisualizer.js

```diff
@@ -185,6 +186,38 @@ LeafletGeomVisualizer.prototype.update = function(time) {
     return true;
 };
 
+/**
+ * Computes the rectangular bounds which encloses the collection of
+ * entities to be visualized.
+ *
+ * @returns {LatLngBounds} The computed bounds.
+ */
+LeafletGeomVisualizer.prototype.getLatLngBounds = function() {
+
+    var bounds = [];
+
+    Object.keys(this._entityHash).forEach(entityId => {
+        const entityDetails = this._entityHash[entityId];
+
+        Object.keys(entityDetails).forEach(primitiveId => {
+            const primitive = entityDetails[primitiveId];
+
+            if (defined(primitive.layer)) {
+                if (defined(primitive.layer.getBounds)) {
```

Comment:
It'd be more beneficial here to use `extend` rather than putting all the coordinates and bounds into an array and combining them, cause the array could be big-ish with a lot of points.